Resources/Prototypes/Recipes/Lathes/medical.yml

```diff
@@ -191,9 +191,19 @@
     Plastic: 300
 
 - type: latheRecipe
+<<<<<<< HEAD
```

Comment:
This is a merge conflict marker left behind in the file. This is unfortunately not the way to resolve a merge conflict. It seems there's a lot going on in this PR now that doesn't quite make sense. Why is HolosignSystem being deleted, for example?